appveyor.yml

```diff
@@ -1,28 +1,51 @@
 version: v1.11.0.build{build}
 
+image:
```

Comment:
I'm not sure we want this. AppVeyor is just some experimental builds we distribute. Can you please drop these changes and let's discuss them in a follow-up PR, since they are not related to getting the build going.